src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -68,14 +65,15 @@ class JSONSerializer(Serializer):
     +-------------------------------------+----------+----------------------------------------+
     |                           |          | Callable(SerializationContext, str) -> str       |
     |                           |          |                                                  |
-    | ``subject.name.strategy`` | callable | Instructs the JsonSerializer on how to construct |
-    |                           |          | Schema Registry subject names.                   |
+    | ``subject.name.strategy`` | callable | Defines how Schema Registry subject names are    |
```

Comment:
Perhaps note how this callable should behave. E.g., return one of these strings: "inherit", "foo", ..

Reply:
i will note the namespace where standard naming strategies are defined (generally shouldn't use a custom one)